### PR TITLE
Add Rint as activation function

### PR DESCRIPTION
**Pull request by Atticus1806**
Adds "tf.math.rint" for rounding as activation function.

### CODE REVIEW

File: nn/math_.py [hunk 1]
```diff
@@ -41,6 +41,11 @@ def floor(x: nn.Tensor) -> nn.Tensor:
   return _activation(x, activation="floor")
 
 
+def rint(x: nn.Tensor) -> nn.Tensor:
+  """round"""
```
Comment (albertz) [inline nn/math_.py:45]: The docstring is misleading. rint is not round, although very similar. round is another separate function.

Comment (Atticus1806) [reply nn/math_.py:45]: oh true, had it on round first. Will change

**Comment by Atticus1806** [comment]
So I think `rint` came from `numpy` which also calls it `rint`, but at the same time someone implemented `round` in `tf`. From my understanding, in general `round` will always do "bankers rounding", while `rint` current system rounding mode, which defaults to the same.
For me it does not really matter, decided for `rint` because its closer to `numpy` and also does not shadow `round`.

In general (not the way tensorflow implements) `round` rounds to given number of decimals, while `rint` is always to next int.

**Comment by Atticus1806** [comment]
I think Pytorch only has `round`

**Pull request merged**